addons/website_event/static/src/js/display_timer_widget.js

```diff
@@ -17,7 +18,7 @@ publicWidget.registry.displayTimerWidget = publicWidget.Widget.extend({
     start: function () {
         var self = this;
         return this._super.apply(this, arguments).then(function () {
-            self.options = self.$el.data();
+            self.options = getElementData(self.el);
```

Comment:
Actually, we have cherry-picked the commit. This is not the only place where we need `getElementData`. jQuery returns the formatted dataset by default in `.data()`, while in JavaScript it returns key-value pairs in string format. To avoid redundant computations of type conversion, we created that utility. But anyway, let's avoid it here for now.